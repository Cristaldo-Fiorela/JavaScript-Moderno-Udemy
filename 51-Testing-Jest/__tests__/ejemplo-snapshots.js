const cliente = {
    nombre: 'Fiorela',
    balance: 500,
    tipo: 'Premium',
};

// * Snapshot
// Permite almacenar datos en una base de datos interna (crea su carpeta con archivo)
// Este almacenamiento permite comparar con aquello que estas evaluando
// Evalua todo el objeto, sin necesidad de ingresar a sus propiedades mediante el .
// Para actualizar nuestro cliente en el archivo de snapshot debemos escribir en nuestra consola "npm t -- -u"

describe('Testing al cliente', () => {
    test('Es Fiorela', () => {
        expect(cliente).toMatchSnapshot('Fiorela');
    })
});