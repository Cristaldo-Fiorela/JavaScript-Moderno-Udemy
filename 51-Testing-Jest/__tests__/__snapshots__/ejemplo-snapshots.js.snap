// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Fiorela: Fiorela 1`] = `
{
  "balance": 500,
  "nombre": "Fiorela",
  "tipo": "Premium",
}
`;
